Description: Fix wrong search engine logging from yahoo sites.
Author: <azhrarn@underhanded.org>
Bug-Debian: http://bugs.debian.org/347426

--- a/wwwroot/cgi-bin/lib/search_engines.pm
+++ b/wwwroot/cgi-bin/lib/search_engines.pm
@@ -364,7 +364,7 @@
 'google\.'=>'translate\.google\.',
 'msn\.'=>'hotmail\.msn\.',
 'tiscali\.'=>'mail\.tiscali\.',
-'yahoo\.'=>'mail\.yahoo\.',
+'yahoo\.'=>'(?:picks|mail)\.yahoo\.|yahoo\.[^/]+/picks',
 'yandex\.'=>'direct\.yandex\.'
 );
 
