Description: Report permissions problem while reading awstats.custom.conf
Author: Ken Neighbors <KenNeighbors@gmail.com>
Bug-Debian: http://bugs.debian.org/572353

--- a/wwwroot/cgi-bin/awstats.pl
+++ b/wwwroot/cgi-bin/awstats.pl
@@ -1750,11 +1750,17 @@
 			$FileSuffix = ".$SiteConfig";
 			last;
 		}
+		elsif ( -e "$searchdir$PROG.$SiteConfig.conf" ) {
+			error("Couldn't open config file \"$searchdir$PROG.$SiteConfig.conf\": $!" );
+		}
 		if ( open( CONFIG, "$searchdir$PROG.conf" ) ) {
 			$FileConfig = "$searchdir$PROG.conf";
 			$FileSuffix = '';
 			last;
 		}
+		elsif ( -e "$searchdir$PROG.conf" ) {
+			error("Couldn't open config file \"$searchdir$PROG.conf\": $!" );
+		}
 	}
 	if ( !$FileConfig ) {
 		error(
